### PR TITLE
Hook Implement

### PR DESCRIPTION
**Pull request by Shaasho**

### CODE REVIEW

File: Where_Is_My_Wife/Assets/Hook_QTE/Scripts/HookUIBar.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+using WhereIsMyWife.Controllers;
+using WhereIsMyWife.Managers;
+
+public interface IHookUIEvents
+{
+    Action<bool> QTEStateEvent { get; set; }
+    Action QTETimeExpired { get; set; }
+}
+
+public class HookUIBar : Singleton<HookUIBar>, IHookUIEvents
```
Comment (Horothenic) [inline Where_Is_My_Wife/Assets/Hook_QTE/Scripts/HookUIBar.cs:12]: All classes whould be in a namespace, in this case ´WhereIsMyWife.UI´ looks appropriate.

File: Where_Is_My_Wife/Assets/_Main/Scripts/Player/Manager/PlayerManager.cs [hunk 2]
```diff
@@ -1,5 +1,7 @@
 using System;
+using System.Security.Cryptography;
```
Comment (Horothenic) [inline Where_Is_My_Wife/Assets/_Main/Scripts/Player/Manager/PlayerManager.cs:2]: Not needed

File: Where_Is_My_Wife/Assets/_Main/Scripts/Player/Manager/PlayerManager.cs [hunk 3]
```diff
@@ -1,5 +1,7 @@
...
 using UnityEngine;
+using UnityEngine.UIElements;
```
Comment (Horothenic) [inline Where_Is_My_Wife/Assets/_Main/Scripts/Player/Manager/PlayerManager.cs:4]: Not needed

**Review by Horothenic — CHANGES_REQUESTED** [review]
Minor things :D

File: Where_Is_My_Wife/Assets/Hook_QTE/Scripts/HookUIBar.cs (hunk 1 above)
Comment (Shaasho) [reply Where_Is_My_Wife/Assets/Hook_QTE/Scripts/HookUIBar.cs]: 902241d025ae61ebfb8e4b08abd978123a88dfcb

File: Where_Is_My_Wife/Assets/_Main/Scripts/Player/Manager/PlayerManager.cs (hunk 2 above)
Comment (Shaasho) [reply Where_Is_My_Wife/Assets/_Main/Scripts/Player/Manager/PlayerManager.cs]: d9e23ebc9860c049fb11139ec743372b7cce66ba

File: Where_Is_My_Wife/Assets/_Main/Scripts/Player/Manager/PlayerManager.cs (hunk 3 above)
Comment (Shaasho) [reply Where_Is_My_Wife/Assets/_Main/Scripts/Player/Manager/PlayerManager.cs]: d9e23ebc9860c049fb11139ec743372b7cce66ba

**Review by Horothenic — APPROVED** [review]

**Review by Horothenic — APPROVED** [review]

**Pull request merged**